### PR TITLE
Dprice storage release 07 26

### PR DESCRIPTION
**Pull request by danieltprice**
- Release note:
https://neon-next-git-dprice-storage-release-07-26-neondatabase.vercel.app/docs/release-notes/2023-07-26-storage-and-compute
- Added an Autoscaling guide:
https://neon-next-git-dprice-storage-release-07-26-neondatabase.vercel.app/docs/guides/autoscaling-guide

Other minor related revisions.
Removed Autoscaling "Beta" tag from docs and Pricing page on the website.
Minor unrelated edits.

### CODE REVIEW

File: content/docs/introduction/autoscaling.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
 title: Autoscaling
-subtitle: Learn how Neon automatically and transparently scales compute on demand
+subtitle: Learn how Neon automatically scales compute resources on demand
 enableTableOfContents: true
 ---
 
-A Beta version of Neon's _Autoscaling_ feature is now available for [Pro plan](/docs/introduction/pro-plan) users in selected regions. The Autoscaling feature dynamically adjusts the allocation of vCPU and RAM for a Neon compute endpoint in response to the current load, eliminating the need for manual intervention.
+A Beta version of Neon's _Autoscaling_ feature is now available for [Pro plan](/docs/introduction/pro-plan) users. The Autoscaling feature dynamically adjusts the amount of compute resources allocated to a Neon compute endpoint in response to the current load, eliminating the need for manual intervention.
```
Comment (vadim2404) [inline content/docs/introduction/autoscaling.md:7]: we discussed with @stepashka to remove the beta label

File: content/docs/guides/autoscaling-guide.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    <Admonition type="note">
+    You can also specify a default **Auto-suspend delay** setting at project creation time. The **Auto-suspend delay** setting defines the number of seconds of inactivity after which a compute endpoint is automatically suspended. This feature is also referred to as "scale-to-zero". The setting specified at project creation time becomes the default setting for newly created compute endpoints.
+    </Admonition>
+
+6. Click **Create Project**. Your initial compute endpoint is created with the specified settings. All compute endpoints created in the future when creating a branch or adding a compute endpoint to a branch are created with the settings you specified.
+
+## Monitor Autoscaling
+
+The `neon_utils` extension provides a `num_cpus()` function for monitoring how the _Autoscaling_ feature allocates compute resources in response to workload. For more information, see [The neon_utils extension](/docs/extensions/neon-utils).
```
Comment (vadim2404) [inline content/docs/guides/autoscaling-guide.md:57]: do you warn about fractional CUs here as well?

File: content/docs/introduction/autoscaling.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/introduction/autoscaling.md]: Thank you. Renmoved.

File: content/docs/guides/autoscaling-guide.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/guides/autoscaling-guide.md:57]: I think we're OK to leave that detail in the neon-utils page. User will have to go there to find out how to use it anyway.

**Review by vadim2404 — APPROVED** [review]

**Pull request merged**